### ISSUE TITLE
Environment variables do not automatically load from .env files

### ISSUE THREAD

**Issue by johnnymcmike** [KSU-Google-DSC/Webapp#49]
According to https://nextjs.org/docs/basic-features/environment-variables#default-environment-variables, nextjs is supposed to automatically load default environment variables from files named things like .env.development. As per the output log, it seems to be doing just that, but when attempting to query the database, the application gives an error suggesting that it hasn't.

Upon starting the app:
```
npm run dev

> src@0.1.0 dev
> next dev

ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Loaded env from C:\Users\mbjmc\Documents\projects\Webapp\src\.env.development
warn  - You have enabled experimental feature (esmExternals) in next.config.js.
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.
event - compiled client and server successfully in 2.3s (1095 modules)
wait  - compiling / (client and server)...
event - compiled client and server successfully in 501 ms (1100 modules)
wait  - compiling /api/auth/[...nextauth] (client and server)...
event - compiled successfully in 52 ms (51 modules)
```
All seems well - note that "info - Loaded env" line. But when doing something that requires querying the database, this server error is raised:
```
error - FirebaseAppError: Service account object must contain a string "project_id" property.
    at new ServiceAccount (C:\Users\mbjmc\Documents\projects\Webapp\src\node_modules\firebase-admin\lib\app\credential-internal.js:136:19)
    at new ServiceAccountCredential (C:\Users\mbjmc\Documents\projects\Webapp\src\node_modules\firebase-admin\lib\app\credential-internal.js:70:15)
    at cert (C:\Users\mbjmc\Documents\projects\Webapp\src\node_modules\firebase-admin\lib\app\credential-factory.js:103:54)
    at eval (webpack-internal:///./database.js:9:73)
    at Object../database.js (C:\Users\mbjmc\Documents\projects\Webapp\src\.next\server\pages\events.js:32:1)
    at __webpack_require__ (C:\Users\mbjmc\Documents\projects\Webapp\src\.next\server\webpack-runtime.js:33:42)
    at eval (webpack-internal:///./pages/api/events/all.js:6:67)
    at Object../pages/api/events/all.js (C:\Users\mbjmc\Documents\projects\Webapp\src\.next\server\pages\events.js:42:1)
    at __webpack_require__ (C:\Users\mbjmc\Documents\projects\Webapp\src\.next\server\webpack-runtime.js:33:42)
    at eval (webpack-internal:///./pages/events.js:13:73) {
  errorInfo: {
    code: 'app/invalid-credential',
    message: 'Service account object must contain a string "project_id" property.'
  },
  codePrefix: 'app',
  page: '/events'
}
```
Don't know how else to include this information, but the fancy error page in-browser shows that this happens when creating the cert object out of the environment variables, in database.js at line 4. The error mentioning that it needs a string property seems to imply that it isn't getting what it needs from the environment variables. On a local version using dotenv to explicitly load environment variables (before [this commit](https://github.com/KSU-Google-DSC/Webapp/commit/a720b98edb1051ad6db2175687c26b7b1880465a) was added), the same error was occurring, but stopped when I added a line calling `dotenv.config(...)`, which is the method call that actually loads things in from .env files.

So, in short, the output log seems at first glance to be lying about having loaded in data from .env.development, or perhaps they're somehow being put someplace other than process.env or being given different names.

Steps To Reproduce:
-checkout dev
-run `npm ci`
-start application with `npm run dev`
-navigate to localhost:3000/events

**Comment by teamrocket77** [comment]
Tried running this and ran into the issue.
Looked at [docs](https://firebase.google.com/docs/reference/js/v8/firebase#initializeapp).
I saw that they were jus passing json into the initializeApp...
removed lines new file looked like the below. 
```
import { initializeApp, cert, getApps, getApp } from 'firebase-admin/app';

export let firestoreApp = !getApps().length ? initializeApp({
    projectId: process.env.FIREBASE_SERVICE_ACCOUNT_PROJECT_ID,
    privateKey: process.env.FIREBASE_SERVICE_ACCOUNT_PRIVATE_KEY,
    clientEmail: process.env.FIREBASE_SERVICE_ACCOUNT_CLIENT_EMAIL,
  databaseURL: process.env.FIRESTORE_SERVICE_DATABASE_URL
}, "COMMON") 
: getApp("COMMON");
```
 Then I started getting this error
<img width="716" alt="Screen Shot 2022-10-21 at 3 49 54 AM" src="https://user-images.githubusercontent.com/14345179/197142520-9152a585-c108-44b2-b69d-8c0c59780494.png">
did a brief search and say that it happens when [GCP rejects refresh token](https://groups.google.com/g/adwords-api/c/VR6vKQBYtEk)
Is it possible that this is the issue?

**Comment by Sanjay-B** [comment]
> Tried running this and ran into the issue. Looked at [docs](https://firebase.google.com/docs/reference/js/v8/firebase#initializeapp). I saw that they were jus passing json into the initializeApp... removed lines new file looked like the below.
> 
> ```
> import { initializeApp, cert, getApps, getApp } from 'firebase-admin/app';
> 
> export let firestoreApp = !getApps().length ? initializeApp({
>     projectId: process.env.FIREBASE_SERVICE_ACCOUNT_PROJECT_ID,
>     privateKey: process.env.FIREBASE_SERVICE_ACCOUNT_PRIVATE_KEY,
>     clientEmail: process.env.FIREBASE_SERVICE_ACCOUNT_CLIENT_EMAIL,
>   databaseURL: process.env.FIRESTORE_SERVICE_DATABASE_URL
> }, "COMMON") 
> : getApp("COMMON");
> ```
> 
> Then I started getting this error <img alt="Screen Shot 2022-10-21 at 3 49 54 AM" width="716" src="https://user-images.githubusercontent.com/14345179/197142520-9152a585-c108-44b2-b69d-8c0c59780494.png"> did a brief search and say that it happens when [GCP rejects refresh token](https://groups.google.com/g/adwords-api/c/VR6vKQBYtEk) Is it possible that this is the issue?

Possibly. I'll investigate further later today. @teamrocket77

**Comment by teamrocket77** [comment]
Most of conversation has occurred offline.
However @johnnymcmike has said the new file provided works.
file was in the form of 
`key: value` and should've been in the format
`key="value"`
